### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by mlloreda**
Improved readability and formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +13,24 @@ Several of ArrayFire's benefits include:
 * Commercial support from [ArrayFire](http://arrayfire.com)
 * [Read about more benefits on Arrayfire.com](http://arrayfire.com/the-arrayfire-library/)
 
-### Build and Test Status
+ArrayFire provides software developers with a high-level
+abstraction of data which resides on the accelerator, the `af::array` object.
+Developers write code which performs operations on ArrayFire arrays which, in turn,
+are automatically translated into near-optimal kernels that execute on the computational
+device.
```
Comment (pavanky) [inline README.md]: Check if a paragraph break here makes it look better?

File: README.md [hunk 2]
```diff
@@ -24,14 +13,24 @@ Several of ArrayFire's benefits include:
...
+ArrayFire is successfully used on devices ranging from low-power mobile phones to
+high-power GPU-enabled supercomputers including CPUs from all major vendors (Intel, AMD, Arm),
```
Comment (pavanky) [inline README.md]: ARM

File: README.md [hunk 3]
```diff
@@ -24,14 +13,24 @@ Several of ArrayFire's benefits include:
...
+GPUs from the dominant manufacturers (NVIDIA, AMD, and Qualcomm), as well as a variety
```
Comment (pavanky) [inline README.md]: Prominent instead of dominant.

File: README.md (hunk 2 above)
Comment (pavanky) [inline README.md]: Can you see if this sentence can be re-written ?

**Comment by mlloreda** [comment]
@pavanky all issues have been addressed.

**Comment by mlloreda** [comment]
skip arrayfire linux ci
skip arrayfire windows ci
skip arrayfire osx ci

**Comment by pavanky** [comment]
@mlloreda What's happening here?

**Comment by mlloreda** [comment]
@pavanky unless there are any more suggestions this is ready to be merged.

**Pull request merged**